manifest/provider/apply.go

```diff
@@ -440,7 +440,7 @@ func (s *RawProviderServer) ApplyResourceChange(ctx context.Context, req *tfprot
 				if err == context.DeadlineExceeded {
 					resp.Diagnostics = append(resp.Diagnostics,
 						&tfprotov5.Diagnostic{
-							Severity: tfprotov5.DiagnosticSeverityError,
+							Severity: tfprotov5.DiagnosticSeverityWarning,
 							Summary:  "Operation timed out",
 							Detail:   "Terraform timed out waiting on the operation to complete",
 						})
```

Comment:
I would perhaps change this message to say something about how the resource has been created but timed out on waiting for whatever is in their `wait` block.